src/DynamoCore/Search/SearchElements/NodeSearchElement.cs

```diff
@@ -278,6 +278,9 @@ protected virtual IEnumerable<string> GenerateOutputParameters()
         }
     }
 
+    /// <summary>
+    /// This class is being used when drag-drop search element on canvas to create a node.
```

Comment:
This class returns (use cref) NodeSearchElement which is used for creating a node.